authors/apps/comments/signals.py

```diff
@@ -18,3 +21,21 @@ def on_comment_post_save(sender, **kwargs):
     if kwargs['created']:
         comment_published_signal.send(CommentsSignalSender,
                                       comment=kwargs['instance'])
+
+
+@receiver(m2m_changed, sender=Comment.liked_by.through)
```

Comment:
hat does the `m2m_changed` mean? If I am asking this, it means readability is a problem.

Reply:
@pbkabali, a question and/or feedback for you to consider

Reply:
@malep2007 this is an inbuilt django model signal that is sent everytime a many-to-many field is changed by an instance. Came across it when doing some research with @oma0256 about sending notifications.

Reply:
So, is it possible to provide a better name for it? One that is more intuitive. This is its usage in a generic sense.

Reply:
Yes it is. probably import it as a different name and write some comments

Reply:
This has been done. Thanks for the feedback @malep2007